api/server/router/system/system.go

```diff
@@ -9,25 +9,27 @@ import (
 // systemRouter provides information about the Docker system overall.
 // It gathers information about host, daemon and container events.
 type systemRouter struct {
-	backend Backend
-	cluster ClusterBackend
-	routes  []router.Route
-	fscache *fscache.FSCache // legacy
-	builder *buildkit.Builder
+	backend         Backend
+	cluster         ClusterBackend
+	routes          []router.Route
+	fscache         *fscache.FSCache // legacy
+	builder         *buildkit.Builder
+	featureBuildKit *bool
 }
 
 // NewRouter initializes a new system router
-func NewRouter(b Backend, c ClusterBackend, fscache *fscache.FSCache, builder *buildkit.Builder) router.Router {
+func NewRouter(b Backend, c ClusterBackend, fscache *fscache.FSCache, builder *buildkit.Builder, featureBuildKit *bool) router.Router {
```

Comment:
Not a fan of the `featureBuildKit` boolean; can't we make this a `features []string` or `features map[string]string` ?